src/functionalTest/java/uk/gov/hmcts/reform/pcqloader/postdeploy/PcqLoaderFunctionalTest.java

```diff
@@ -48,6 +49,8 @@ public class PcqLoaderFunctionalTest extends PcqLoaderTestBase {
     @Autowired
     private BlobStorageManager blobStorageManager;
 
+    @Value("${telemetry.wait.period:10000}")
```

Comment:
Can you rename this to use something like the following. Doesn't matter the property doesn't exist.
'functional-test.wait.period:10000'

Reply:
Thanks Dave will do the changes